packages/cli/src/services/check-parser/parser.ts

```diff
@@ -41,7 +47,7 @@ const supportedBuiltinModules = [
 
 function validateEntrypoint (entrypoint: string): {extension: SupportedFileExtension, content: string} {
   const extension = path.extname(entrypoint)
-  if (extension !== '.js' && extension !== '.ts') {
+  if (extension !== '.js' && extension !== '.ts' && extension !== '.mjs') {
```

Comment:
```suggestion
  if (!['.js', '.ts', '.mjs'].includes(extension)) {
```

Reply:
@umutuzgur , I tried but there is a TS complaint because the transpiler doesn't understand that the `extensions` filtered values are all within the `SupportedFileExtension` values:

![image](https://github.com/checkly/checkly-cli/assets/5315705/1e2c7790-6725-4392-b759-e4c21a9a4caa)